### ISSUE TITLE
Remote Support Insight Online error

### ISSUE THREAD

**Issue by hbxx** [HewlettPackard/POSH-HPOneView#497]
when issuing the following command:

```powershell
 Set-OVRemoteSupport -CompanyName $config.support.company -InsightOnlinePassword $Insight.password -InsightOnlineUsername $Insight.Username -OptimizeOptIn
```
i receive the following error:
```powershell
Send-OVRequest : Cannot validate argument on parameter 'body'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
At C:\Program Files\WindowsPowerShell\Modules\HPEOneView.530\5.30.2472.1534\HPEOneView.530.psm1:24743 char:119
+ ... talRegistraionUri -Method POST -Body $_PortalRestrationObject -Hostna ...
+                                          ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Send-OVRequest], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Send-OVRequest
```
it appears the body variable is misspelled (missing 'g'). It should be `$_PortalRegsitrationObject`

**Comment by ChrisLynchHPE** [comment]
This will be fixed in the next 5.20 and 5.30 library updates.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.30.2515.1313](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.30.2515.1313) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.530/5.30.2515.1313) and [Release 5.20.2515.1660](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.20.2515.1660) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.520/5.20.2515.1660).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
